mqtt5/README.md

```diff
@@ -2,6 +2,8 @@
 
 This document defines how to describe MQTT 5-specific information on AsyncAPI.
 
+**Deprecation Warning**: MQTT version 5 specific bindings are deprecated in favor of [MQTT bindings](../mqtt/README.md) that are not version specific.
```

Comment:
The deprecation note is not very visible:

![Screenshot 2023-06-26 at 19 40 38](https://github.com/asyncapi/bindings/assets/6995927/fdc2f114-46dc-4547-ad0c-4d764e92f81c)

```suggestion
# **Deprecation Warning**: MQTT version 5 specific bindings are deprecated in favor of [MQTT bindings](../mqtt/README.md) that are not version specific.
```

better make it `h1` and best probably move to the very top of the document

Reply:
@pearmaster please have a look